transitions/extensions/diagrams.py

```diff
@@ -74,7 +74,23 @@ def _add_edges(self, events, container):
                 src = transitions[0]
                 for t in transitions[1]:
                     dst = t.dest
-                    container.add_edge(src, dst, label=label)
+                    lbl = self._transition_label(label, t)
+                    container.add_edge(src, dst, label=lbl)
+
+    @staticmethod
+    def edge_label_from_transition_label(label):
```

Comment:
Moved it.

Reasoning to leave it in AGraph was to get it close to reverse code `_transition_label` as both are making same assumption on formatting. But it doesn't matter really.